lib/reek/examiner.rb

```diff
@@ -63,16 +66,18 @@ def smelly?
     private
 
     def run
-      smell_repository = Smells::SmellRepository.new(description, @smell_types)
+      smell_repository = Smells::SmellRepository.new(source_description: description,
```

Comment:
I've always wondered why SmellRepository had a description parameter. Now finally we have some kind of use for it.